tests/testcases/a_basics/container_testcases.py

```diff
@@ -0,0 +1,106 @@
+from tests.testcases.base_test import BaseTest
+import unittest
+import time
+from random import randint
+from zerorobot.dsl.ZeroRobotManager import ServiceCreateError
+from tests.controller.controller import Controller
+
+class TESTContainer(BaseTest):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def setUp(self):
+        super().setUp()
+
+    def test001_create_containers(self):
+        """ ZRT-ZOS-001
+        *Test case for creatings container on a zero-os node*
+        **Test Scenario:**
+        #. Create a two container on that node, should succeed.
+        #. Check that the container have been created.
+        """
+        self.logger.info('Create a two container on that node, should succeed.')
+        container_1_data = self.get_container_default_data()
+        self.controller.container_manager.install(wait=True, data=container_1_data)
+        self.assertTrue(self.controller.container_manager.install_state, " Installtion state is False")
+        self.controller_2 = Controller(config=self.config, god_token=None)        
+        container_2_data = self.get_container_default_data()
+        self.controller_2.container_manager.install(wait=True, data=container_2_data)
+        self.assertTrue(self.controller_2.container_manager.install_state, " Installtion state is False")
+    
+        self.log('Check that the container have been created.')
+        conts = self.controller.node.client.container.list()
+        self.assertTrue([c for c in conts.values() if c['container']['arguments']['hostname'] == container_1_data['hostname']])
+        self.assertTrue([c for c in conts.values() if c['container']['arguments']['hostname'] == container_2_data['hostname']])
+
+
+    def test002_create_container_with_all_possible_params(self):
```

Comment:
after reviewing this test case, its only check `env` params, It doesnt check `mount`, `storage` .... so please rename it and create another test cases to validate those params